org.oasis.xdita/dtd/lw-topic.mod

```diff
@@ -140,168 +135,132 @@ PUBLIC "-//OASIS//ELEMENTS LIGHTWEIGHT DITA Topic//EN"
 <!--                    ELEMENT DECLARATIONS                       -->
 <!-- ============================================================= -->
 
-<!--                    LONG NAME: Alternative content  -->
-<!ELEMENT alt           (#PCDATA|%ph;|%data;)*        >
-<!ATTLIST alt
-             %localization;
-             %filters;
-             %variable-content;
-             outputclass  CDATA          #IMPLIED
-             class CDATA "- topic/alt ">
-             
-<!--                    LONG NAME: Body                  -->
-<!ELEMENT body          ((%list-blocks;)*, section*, fn*)        >
-<!ATTLIST body
+<!--                    LONG NAME: Topic  -->
+<!ELEMENT topic   (title, shortdesc?, prolog?, body?)  >
+<!ATTLIST topic
+             id       ID          #REQUIRED
+             xmlns:ditaarch CDATA #FIXED "http://dita.oasis-open.org/architecture/2005/"
+	         ditaarch:DITAArchVersion CDATA "2.0"
+             specializations CDATA "&included-domains;"
+             outputclass  CDATA    #IMPLIED
              %localization;
-             outputclass  CDATA          #IMPLIED
-             class CDATA "- topic/body ">
-             
-<!--                    LONG NAME: Data  -->
-<!ELEMENT data             (#PCDATA|%data;)*        >
-<!ATTLIST data
+             class CDATA "- topic/topic ">
+
+<!--                    LONG NAME: Title -->
+<!ELEMENT title (%common-inline;)* >
+<!ATTLIST title
              %localization;
-             name       CDATA                            #IMPLIED
-             value      CDATA                            #IMPLIED
-             %reference-content;
-             %variable-content;
-             %filters;
              outputclass  CDATA          #IMPLIED
-             class CDATA "- topic/data ">             
-             
-<!--                    LONG NAME: Description   -->
-<!ELEMENT dd             (%list-blocks;)*        >
-<!ATTLIST dd
+             class CDATA "- topic/title ">
+
+<!--                    LONG NAME: Short description-->
+<!ELEMENT shortdesc     (%all-inline;)* >
+<!ATTLIST shortdesc
              %localization;
              %filters;
              %reuse;
              outputclass  CDATA          #IMPLIED
-             class CDATA "- topic/dd ">      
-             
-<!--                    LONG NAME: Description  -->
-<!ELEMENT desc		(%common-inline;)*        >
-<!ATTLIST desc
+             class CDATA "- topic/shortdesc ">
+
+<!--                    LONG NAME: Prolog-->
+<!ELEMENT prolog (%data;)* >
+<!ATTLIST prolog
              %localization;
              %filters;
-             outputclass  CDATA          #IMPLIED
-             class CDATA "- topic/desc ">             
+             class CDATA "- topic/prolog ">
 
-<!--                    LONG NAME: Description list -->
-<!ELEMENT dl             (dlentry)+       >
-<!ATTLIST dl
+
+<!--                    LONG NAME: Body                  -->
+<!ELEMENT body          ((%list-blocks;)*, section*, fn*)        >

```

Comment:
The base DITA model does not allow fn inside of body so this content model is not compatible